fmn/backends/pagure.py

```diff
@@ -120,8 +123,8 @@ async def get_group_projects(
         else:
             params_seq = [
                 {"projects": True, "acl": role.name.lower()}
-                for role in PagureRole
-                if role & acl and role.bit_count() == 1
+                for role in PagureRole.GROUP_ROLES
```

Comment:
Ah, that's what I read before in earlier version of your PR: you can't iterate over enum items built with `|` before Python 3.11.

Reply:
Ahh…, I’ll add a workaround.